bot.py

```diff
@@ -167,14 +186,20 @@ def create_wbw_session():
 
 
 def wbw_get_lowest_member(voted):
-    """Looks up the wiebetaaltwat balance and returns the slack name of the lowest standing balance holder
+    """
```

Comment:
The first line of a docstring should start on the same line as the opening """

Reply:
From pep8:
> For flowing long blocks of text with fewer structural restrictions (docstrings or comments), the line length should be limited to 72 characters.

The line is 73 chars (including the space after ",").